addons/hr/models/hr_employee.py

```diff
@@ -113,6 +113,11 @@ def _default_image(self):
     departure_description = fields.Text(string="Additional Information", groups="hr.group_hr_user", copy=False, tracking=True)
     departure_date = fields.Date(string="Departure Date", groups="hr.group_hr_user", copy=False, tracking=True)
     message_main_attachment_id = fields.Many2one(groups="hr.group_hr_user")
+    job_title = fields.Char(compute="_compute_job_title", store=True, readonly=False, copy=True)
+    work_phone = fields.Char(compute="_compute_phones", store=True, readonly=False, copy=True)
```

Comment:
Following the guidelines, the compute method should be named ``_compute_work_phone``.

This comment probably applies to other fields on both PR.

Reply:
`work_phone` and `mobile_phone` are computed in the same method.
Should I call it `_compute_work_and_mobile_phone` or `_compute_from_address_id` or separate it in 2 methods ?

Reply:
Let's keep it `_compute_phones` then ;)